src/components/notifications/notifications.tsx

```diff
@@ -57,6 +57,9 @@ export class Notifications extends React.Component<NotificationsProps> {
                 pauseOnFocusLoss: true,
                 draggable: false,
                 autoClose: AUTO_CLOSE_TIMEOUT,
+                style: {
+                    wordBreak: "break-all"
```

Comment:
```suggestion
                    overflowWrap: "break-word"
```

[`break-word`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) might be more legible. but test it out and make sure

Reply:
[Looks like](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#comparing_overflow-wrap_word-break_and_hyphens) the main difference is that it will add a newline before the long word. I'm honestly not sure if that's more legible or not 😅

Reply:
> [Looks like](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#comparing_overflow-wrap_word-break_and_hyphens) the main difference is that it will add a newline before the long word. I'm honestly not sure if that's more legible or not 😅
> 
> I think it would avoid the "denie-d" in your screenshot and would make it on one line though? That would be better for shorter words that _could_ fit on one line then

Thanks for your suggestion, you are correct, it is much better to use `break-word` instead of `break-all` so that it only breaks the long words.
But I found it is only working with `word-break: break-word;` and it is not working with `overflow-wrap: break-word`, I found someone is facing the same issue with this component, I can change my PR to `word-break: break-word;` if it is good to you.
https://github.com/fkhadra/react-toastify/issues/112

After using `word-break: break-word;`:
<img width="322" alt="image" src="https://github.com/argoproj/argo-ui/assets/101035990/ee9aaa98-a108-407d-a528-6e869d156682">

Reply:
`word-break: break-word` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#browser_compatibility). It appears to still be supported, but could be removed from browsers, so is not necessarily stable. That's why I suggested `overflow-wrap: break-word`, which is supported.

Strange that it didn't work for you, I wonder if there's some other CSS needed on the `display` or `overflow`? 🤔 
Or if it was overridden by something in the existing CSS?

Another option is `overflow-x: scroll`, which is not the nicest, but works relatively well with touch pads at least.

Reply:
> `word-break: break-word` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#browser_compatibility). It appears to still be supported, but could be removed from browsers, so is not necessarily stable. That's why I suggested `overflow-wrap: break-word`, which is supported.
> 
> Strange that it didn't work for you, I wonder if there's some other CSS needed on the `display` or `overflow`? 🤔 Or if it was overridden by something in the existing CSS?
> 
> Some alternatives include:
> 
> * `overflow-x: scroll`, which is not the nicest, but works relatively well with touch pads at least.
> * [`min-content`](https://developer.mozilla.org/en-US/docs/Web/CSS/min-content) sizing or similar, which would stretch the notification, potentially too much

Hi @agilgur5 
I did some changes to use recommended css override way to fix this issue, could you please help to review?

Reply:
I'd prefer to use `word-break: break-word;` than overriding the component's styles, it is less complicated.

But a simpler way to achieve this is to just add `max-width: 240px` in addition to the `overflow-wrap: 'break-word';`. You can put them on the `div` itself, instead of passing to the `toastMethod`.

I did confirm this works:
![Screenshot 2024-06-10 at 11 06 08 PM](https://github.com/argoproj/argo-ui/assets/4970083/bfeff07e-2951-49da-98f0-bc790cddedca)

Since the toast's width is never changed, we can reliably set a fixed value for `max-width`. (although increasing the width of the toast might be a good idea for this problem)